{
  try {
    m_debugger.resume();
  }
 catch (  final DebugExceptionWrapper e) {
    throw new DebugException(e);
  }
}
